src/radio-group/tests/middleware.spec.ts

```diff
@@ -0,0 +1,55 @@
+const { assert } = intern.getPlugin('chai');
+const { describe, it, afterEach } = intern.getInterface('bdd');
+import cacheMiddleware from '@dojo/framework/core/middleware/cache';
+import icacheMiddleware from '@dojo/framework/core/middleware/icache';
+import { radioGroup as radioGroupMiddleware } from '../middleware';
+import { sandbox } from 'sinon';
+
+const sb = sandbox.create();
+const onValueStub = sb.stub();
+const { callback } = radioGroupMiddleware();
+
+function cacheFactory() {
+	return cacheMiddleware().callback({
+		id: 'test-cache',
+		properties: () => ({}),
+		children: () => [],
+		middleware: { destroy: sb.stub() }
+	});
+}
+
+function icacheFactory() {
+	return icacheMiddleware().callback({
+		id: 'test-cache',
+		properties: () => ({}),
+		children: () => [],
+		middleware: { cache: cacheFactory(), invalidator: sb.stub() }
+	});
+}
+
+describe('RadioGroup-middleware', () => {
+	afterEach(() => {
+		sb.resetHistory();
+	});
+
+	it('converts checked values to arrays', () => {
+		const radioGroup = callback({
+			id: 'radiogroup-test',
+			middleware: { icache: icacheFactory() },
+			properties: () => ({}),
+			children: () => []
+		})(onValueStub);
```

Comment:
I am not 100% sure, but I think the build is failing because this is expecting the `initialValue` argument